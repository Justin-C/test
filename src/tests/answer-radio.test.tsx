import React from 'react';

import renderer from 'react-test-renderer';
import AnswerRadio from '../components/answer-radio';

describe('checkbox tests', () => {
  it('should render checkbox', () => {
    const tree = renderer
      .create(
        <AnswerRadio
          answerText={'test'}
          handleChange={() => {}}
          isEnabled={true}
          isChecked={true}
          name={'test'}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
