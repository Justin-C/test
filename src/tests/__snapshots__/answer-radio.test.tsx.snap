// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checkbox tests should render checkbox 1`] = `
<label>
  <input
    checked={true}
    disabled={true}
    name="radio"
    onChange={[Function]}
    type="radio"
    value="test"
  />
  <p
    className="question__answer-text text--default"
  >
    test
  </p>
</label>
`;
